docs/Events/bpmn-events/import-render-start.md

```diff
@@ -4,9 +4,31 @@ title: import.render.start
 nav_order: 8
 description: import.render.start event
 permalink: /events/bpmn-events/import-render-start
-last_modified_date: 2020-04-27T17:54:08+0000
+last_modified_date: 2020-09-12T17:54:08+0000
 parent: "BPMN Events"
 grand_parent: "Events"
 ---
 
 # import.render.start
+
+This event is fired by [importXML API](/general/api-reference/bpmn-js/importXML) with data containing `definitions` before the graphical import begins. This allows users to listen whenever a rendering of a diagram is about to be started.
```

Comment:
Phrasing (maybe?): `This allows users to listen for the start of the graphical rendering of diagrams.`